indico/modules/rb/client/js/components/Room.jsx

```diff
@@ -175,9 +179,12 @@ class Room extends React.Component {
           </Card.Description>
         </Card.Content>
         <Card.Content styleName="room-content" extra>
-          <>
-            <Icon name="user" /> {room.capacity || Translate.string('Not specified')}
-          </>
+          <Icon name="user" /> {room.capacity || Translate.string('Not specified')}
+          {showLocation && (
+            <div>
+              <Icon name="map pin" /> {room.locationName || Translate.string('Not specified')}
+            </div>
+          )}
```

Comment:
The room card looks rather broken with the location information added:

![image](https://user-images.githubusercontent.com/179599/213238902-6f234483-46ac-4ba9-8a94-64ef6eb445ca.png)

Reply:
This happens because the class `.room-card` in `Room.module.scss` has a fixed height value of 320px. My solution was removing the height of it and let Semantic UI's `Card.Group` component manage the card's heigh, keeping the same height for all cards of that row and avoiding future problems when adding any other new fields to the room card. Let me know if you prefer other approach 🙂

Reply:
(I added a commit to not show the location there)